packages/core/src/browser/status-bar/status-bar.tsx

```diff
@@ -37,6 +37,7 @@ export interface StatusBarEntry {
     alignment: StatusBarAlignment;
     color?: string;
     className?: string;
+    id?: string;
```

Comment:
Is not it redundant?  we already pass an id here: https://github.com/theia-ide/theia/pull/2526/files#diff-f9ebf8fac439ed099cd352d59b6f2edaR91